spec/openapi.yaml

```diff
@@ -11130,6 +11961,49 @@ components:
             $ref: '#/components/schemas/CustomFieldBasicRequest'
           description: Custom fields assigned to the project which job is created in. Validation on custom fields is done in scope of that request.
       type: object
+    CreateJobFuzzyReportRequest:
+      description: The payload to use when generating a job fuzzy report.
+      properties:
+        contentType:
+          enum:
+            - JOB_CONTENT_TRANSLATION
```

Comment:
It is a wrong content type
They us `name`:
 - TRANSLATION
 - UNAUTHORIZED
 etc.